src/relay_sdk/workflows.py

```diff
@@ -0,0 +1,37 @@
+"Allows a step to run a workflow"
+import json
+import logging
+from typing import Any
+from urllib.parse import quote
+
+from requests import Session
+
+from .util import JSONEncoder
+
+logger = logging.getLogger(__name__)
+
+
+class Workflows:
+    """Use Workflows to run a workflow with parameters"""
+
+    def __init__(self, client: Session) -> None:
+        self._client = client
+
+    def run(self, name: str, parameters: Any = {}) -> None:
```

Comment:
The type of `parameters` should be `Mapping[str, Any]` probably.

Reply:
Done!